models/issues/issue_project.go

```diff
@@ -14,27 +14,24 @@ import (
 
 // LoadProject load the project the issue was assigned to
 func (issue *Issue) LoadProject(ctx context.Context) (err error) {
-	if issue.Project == nil {
-		var p project_model.Project
-		has, err := db.GetEngine(ctx).Table("project").
+	if issue.Projects == nil {
+		err = db.GetEngine(ctx).Table("project").
 			Join("INNER", "project_issue", "project.id=project_issue.project_id").
-			Where("project_issue.issue_id = ?", issue.ID).Get(&p)
-		if err != nil {
-			return err
-		} else if has {
-			issue.Project = &p
-		}
+			Where("project_issue.issue_id = ?", issue.ID).OrderBy("title").
+			Find(&issue.Projects)
 	}
 	return err
 }
 
-func (issue *Issue) projectID(ctx context.Context) int64 {
-	var ip project_model.ProjectIssue
-	has, err := db.GetEngine(ctx).Where("issue_id=?", issue.ID).Get(&ip)
-	if err != nil || !has {
-		return 0
+
+func (issue *Issue) projectIDs(ctx context.Context) []int64 {
+	var ips []int64
+	if err := db.GetEngine(ctx).Table("project_issue").Select("project_id").
+		Where("issue_id=?", issue.ID).Find(&ips); err != nil {
+		return nil
```

Comment:
`err` should not be ignored. I know the previous code should still be corrected.